apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -287,4 +288,24 @@ export const pageRouter = router({
       await startProjectById(ctx.logger, codeBuildId)
       return addedVersionResult
     }),
+
+  updatePageSettings: protectedProcedure.input(pageSettingsSchema).mutation(
+    // TODO: save noIndex and meta to db
+    async ({ input: { pageId, siteId, title, meta, permalink, noIndex } }) => {
+      return db
+        .updateTable("Resource")
+        .where("Resource.id", "=", String(pageId))
+        .where("Resource.siteId", "=", siteId)
+        .where("Resource.type", "in", ["Page", "CollectionPage"])
+        .set({ title, permalink })
+        .returning([
+          "Resource.id",
+          "Resource.type",
+          "Resource.title",
+          "Resource.permalink",
+          "Resource.draftBlobId",
+        ])
```

Comment:
do we need to return so many things? from what I see you only really need the new title and permalink (and even then in your current implementation don't use it at all). might want to avoid overreturning